atacworks/io/bw2h5.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
```

Comment:
since these are not executable standalone now, we can take this shebang line out